### PR TITLE
tutorial page for companies done

### PR DESCRIPTION
**Pull request by malinasdfghjkl**
Please check the text for mistakes or if you wan to add anything.

### CODE REVIEW

File: src/views/CompanyTutorialView/CompanyTutorialView.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          </p>
+          <img src={qrpage} alt="Home page"/>
+        </div>
+
+        <div className="subsubtitles">Side bar</div>
+        <div className="text-img">
+          <p>
+            To navigate the app, use the side bar on the left. The top button
+            takes you to the home page.
+            The next button . The next button takes
```
Comment (yashdhanore) [inline src/views/CompanyTutorialView/CompanyTutorialView.tsx:61]: "the next button" is repeated

File: src/views/CompanyTutorialView/CompanyTutorialView.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { FC } from "react";
+import "../TutorialView/tutorial-view.css";
+import "./company-tutorial-view.css";
+import qrSidebar from "../../assets/Tutorial/qr-code-sidebar.png";
+import qrtype from "../../assets/Tutorial/create-ticket-qr.png";
+import qrpage from "../../assets/Tutorial/generate-qr-page.png";
```
Comment (yashdhanore) [inline src/views/CompanyTutorialView/CompanyTutorialView.tsx:6]: why does the logo on sidebar look weird ; (
Maybe i need to look into this

**Comment by yashdhanore** [comment]
I'm not sure why this is happening but when i switched to this branch login page looked like this for me. Didnt have this issue in my recent branch or the main branch.

image

**Review by DanaIso — COMMENTED** [review]
I get the same issue as Yash. Something went wrong in this branch

**Comment by malinasdfghjkl** [comment]
Oh found the issue! Seems to be a class in login with the same name as one in CreateQRCodeView.

**Comment by malinasdfghjkl** [comment]
This should fix it @yashdhanore & @DanaIso. Does it work for you guys now?

File: src/views/CompanyTutorialView/CompanyTutorialView.tsx (hunk 1 above)
Comment (malinasdfghjkl) [reply src/views/CompanyTutorialView/CompanyTutorialView.tsx]: Thank you! Fixed it now:)

File: src/views/CompanyTutorialView/CompanyTutorialView.tsx (hunk 2 above)
Comment (malinasdfghjkl) [reply src/views/CompanyTutorialView/CompanyTutorialView.tsx:6]: what do you mean look weird? In the photo?

**Comment by DanaIso** [comment]
Yes looks good now! Can you resolve the conflicts? Or maybe Yash knows what parts to include

File: src/views/CompanyTutorialView/CompanyTutorialView.tsx (hunk 2 above)
Comment (yashdhanore) [reply src/views/CompanyTutorialView/CompanyTutorialView.tsx:6]: yeah the logos look shifted to the left

**Comment by yashdhanore** [comment]
Works now!

**Comment by malinasdfghjkl** [comment]
@yashdhanore could you take a look at the merge conflicts? I don't know what parts of it that's still relevant.

**Comment by yashdhanore** [comment]
@malinasdfghjkl I merged the first two commits in another branch and merged the conflicts there.

**Comment by mrporsev** [comment]
@yashdhanore @malinasdfghjkl Can I close this ticket since it's been merged in another PR I assume?

**Comment by yashdhanore** [comment]
@mrporsev yes

**Comment by DanaIso** [comment]
@mrporsev @yashdhanore you commented that only the first two commits are merged. @malinasdfghjkl has made other commits after that. Or am I misunderstanding?

**Comment by yashdhanore** [comment]
@DanaIso  No you're right, the third commit was a fix for the login page issue but it caused issues in qrcode page so I fixed that myself and copy pasted the changes of the fourth commit which were some images. 
But yeah don't close now, let @malinasdfghjkl have a look once and make sure the changes in fourth commit are present in main

**Pull request closed without merging**